ckan/public-midnight-blue/base/scss/_module.scss

```diff
@@ -4,6 +4,8 @@
     }
 
     > .module-content{
+        border-radius: 20px;
+        background-color: $layoutPrimaryBackgroundColor;
```

Comment:
![image](https://github.com/user-attachments/assets/28b10231-5daa-430e-a381-f887e8cb1591)

This change disrupts the organization, group pages and all elements that use `.module-content`. What was this change for?

Reply:
@aleeexgreeen it was added to fix this previous design

![image](https://github.com/user-attachments/assets/5d53836a-589a-42b8-a3de-47a0e389365b)

**and now the changed look like this**

![image](https://github.com/user-attachments/assets/dab47b78-4a5a-4cd6-9a4b-3ba178431b0a)

Reply:
Anyway, if this is not needed, I can easily revert. It was just something I noticed while working on this issue, its never part of the main point of this pr